### PR TITLE
Add CSS for tables

### PR DESCRIPTION
**Pull request by nobodywasishere**
Add CSS for tables that matches code blocks (#11899)

### CODE REVIEW

File: src/compiler/crystal/tools/doc/html/css/style.css
Comment (Sija) [inline src/compiler/crystal/tools/doc/html/css/style.css:280]: Personally, I find (all occurrences of) this change less readable.

Comment (nobodywasishere) [reply src/compiler/crystal/tools/doc/html/css/style.css:280]: I can revert it - it was just the default formatting performed by VS Code.

Comment (Sija) [reply src/compiler/crystal/tools/doc/html/css/style.css:280]: I'd leave it to the core team to decide. The rest of the changes are also quite subjective btw, as they depend heavily on your prettier configuration.

Comment (nobodywasishere) [reply src/compiler/crystal/tools/doc/html/css/style.css:280]: I'm not sure I follow, other than the formatting this just adds table css and allowing splits between namespaces

Comment (Sija) [reply src/compiler/crystal/tools/doc/html/css/style.css:280]: I meant the formatting-related ones.

**Comment by straight-shoota** [comment]
I'd prefer three separate PRs for these changes. They are pretty much unrelated to each other than that some of the same files are touched.

**Review by straight-shoota — APPROVED** [review]

**Pull request merged**
